addons/website_sale/views/templates.xml

```diff
@@ -1875,7 +1875,9 @@
                             <div t-if="product_image._name == 'product.image' and product_image.embed_code" class="d-flex align-items-center justify-content-center h-100 embed-responsive embed-responsive-16by9">
                                 <t t-raw="product_image.embed_code"/>
                             </div>
-                            <div  t-else="" t-field="product_image.image_1920" class="d-flex align-items-center justify-content-center h-100" t-options='{"widget": "image", "preview_image": "image_1024", "class": "product_detail_img mh-100", "alt-field": "name", "zoom": product_image.can_image_1024_be_zoomed and "image_1920", "itemprop": "image"}'/>
+                            <div t-else="" class="d-flex align-items-center justify-content-center h-100" contenteditable="false">
+                                <div t-field="product_image.image_1920" t-options='{"widget": "image", "preview_image": "image_1024", "class": "product_detail_img mh-100", "alt-field": "name", "zoom": product_image.can_image_1024_be_zoomed and "image_1920", "itemprop": "image", "contenteditable": "true"}'/>
```

Comment:
Doesn't this make the image editable even when you're not in edit-mode? I remember we talked about this but I'm not sure anymore. Also, any reason `o_not_editable` doesn't work on the wrapper?

Reply:
I had to use a container at the size of the image around it and make it not editable in order to prevent writing text at the level of the image. As the image is editable, if it is displayed in a bigger container, it will be possible to write text besides the image